src/datatypes/datetime.js

```diff
@@ -623,6 +642,14 @@ class Date {
     return result;
   }
 
+  getPrecisionValue() {
+    const precisionValueMap = new Map();
+    precisionValueMap.set(DateTime.Unit.YEAR, 4);
+    precisionValueMap.set(DateTime.Unit.MONTH, 6);
+    precisionValueMap.set(DateTime.Unit.DAY, 8);
+    return precisionValueMap.get(this.getPrecision());
+  }
```

Comment:
Same here -- I'd recommend initializing a constant map, or you could probably use the same map as DateTime uses.

Reply:
Will do the same here!